bundle/run/job.go

```diff
@@ -317,6 +317,25 @@ func (r *jobRunner) Cancel(ctx context.Context) error {
 	return errGroup.Wait()
 }
 
+func (r *jobRunner) Restart(ctx context.Context, opts *Options) (output.RunOutput, error) {
+	// We don't need to cancel existing runs if the job is continuous and unpaused.
+	// the /jobs/run-now API will automatically cancel any existing runs before starting a new one.
+	continuous := r.job.JobSettings.Continuous
+	if continuous != nil && continuous.PauseStatus == jobs.PauseStatusUnpaused {
```

Comment:
What if it's paused?

Reply:
Paused seems to work exactly like a non-continuous job. New job runs are queued and wait for existing one in the queue to finish.

Added a comment saying as much.